src/split-panel/icons/side-position-refresh.tsx

```diff
@@ -6,17 +6,24 @@ import styles from '../styles.css.js';
 export interface SVGTableRowProps {
   offset: number;
   separator?: boolean;
+  isHeader?: boolean;
 }
 
 export const getClassName = (suffix: string) => styles[`preference-icon-refresh--${suffix}`];
 
-const TableRow = ({ offset, separator = true }: SVGTableRowProps) => {
+const TableRow = ({ offset, separator = true, isHeader }: SVGTableRowProps) => {
   const offsetTop = 0.4482;
   const offsetBottom = 3.4482;
   return (
-    <g transform={`translate(0, ${offset})`} className={getClassName('disabled')}>
+    <g
+      transform={`translate(0, ${offset})`}
+      className={isHeader ? getClassName('column-header') : getClassName('disabled')}
```

Comment:
```suggestion
      className={getClassName(isHeader ? 'column-header': 'disabled')}
```

I think this variation is slightly easier to read, only apply if you agree.

Reply:
Good idea, will update in the next revision